Fixes #1034

attackant opened the pull request:
Fixes #1034

attackant committed:
Refactor function parameters and condition in apple-news.php

The parameters in the function "filter_update_post_metadata" have been refactored for better readability and stricter typing. In addition, the condition to check the count of 'old_value' has been enhanced to prevent potential errors when 'old_value' is not an array.

renatonascalves commented on includes/class-apple-news.php:
```diff
@@ -544,19 +544,19 @@ public function create_default_theme() {
+	 * @see \update_metadata
 	 */
-	public function filter_update_post_metadata( $check, $object_id, $meta_key, $meta_value, $prev_value ) {
+	public function filter_update_post_metadata( ?bool $check, int $object_id, string $meta_key, mixed $meta_value, mixed $prev_value ): ?bool {
```
I'd expect typecasting those variables a big no, right?! Since technically, any plugin can hook into `update_post_metadata` and alter the returned type.

attackant replied:
Good point, I'll remove.

attackant committed:
Removed strict typing in "filter_update_post_metadata" function

The strict typing in the parameters of the function "filter_update_post_metadata" has been removed to offer more flexibility. Simultaneously, the condition which checks the count of 'old_value' has been modified to avoid errors if 'old_value' is not an array.

attackant committed:
Add space
